CentOS boot script update

bbrendon opened the pull request:
Description:
Clear up some confusion. I tested the process and it works.

I'm trying to close this issue: #5656


## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

bbrendon committed:
centos

fabaff commented on source/_docs/installation/centos.markdown:
````diff
@@ -52,22 +52,23 @@ You will need to enable the software collection each time you log on before you
 WantedBy=multi-user.target
 ```
+
+This works because the Python virtual environment was created using the SCL envionrment, thus there is no need to activate SCL.
````
```suggestion
This works because the Python virtual environment was created using the SCL environment, thus there is no need to activate SCL.
```

fabaff commented on source/_docs/installation/centos.markdown:
```diff
@@ -52,22 +52,23 @@ You will need to enable the software collection each time you log on before you
+# %i means the username is derrived from the filename.
+User=%i
+# a python venv for hass exists in /opt/hass/venv
+ExecStart=/opt/hass/venv/bin/hass
```
Can we please keep it close to the [systemd](https://www.home-assistant.io/docs/autostart/systemd/) docs?

```suggestion
ExecStart=/srv/homeassistant/bin/hass
```

bbrendon committed:
Update source/_docs/installation/centos.markdown

Fine with me. I did like having the "venv" in there to make it more obvious as to what is going on.

Co-Authored-By: Fabian Affolter <mail@fabian-affolter.ch>

bbrendon committed:
Update source/_docs/installation/centos.markdown

Co-Authored-By: Fabian Affolter <mail@fabian-affolter.ch>

bbrendon reviewed (commented):
Is there something I need to do to get this done?
